eng/common/scripts/Prepare-Release.ps1

```diff
@@ -0,0 +1,109 @@
+#Requires -Version 6.0
+
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$PackageName,
+    [string]$ServiceDirectory,
+    [string]$ReleaseDate, # Pass Date in the form MM/dd/yyyy"
+    [string]$BuildType # For Java
```

Comment:
Instead of requiring this for Java we should consider setting it the java package properties. I think we can figure out the value there and we can probably just default it there if we don't know the value.

Reply:
That will require a separate work to get it working correctly for all the languages without causing other issues. I have opened an issue to track that.